spring-cloud-contract-tools/spring-cloud-contract-gradle-plugin/src/test/groovy/org/springframework/cloud/contract/verifier/plugin/SampleProjectKotlinSpec.groovy

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2013-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.contract.verifier.plugin
+
+import org.gradle.testkit.runner.BuildResult
+import org.junit.Ignore
+import spock.lang.Stepwise
+
+import static org.gradle.testkit.runner.TaskOutcome.UP_TO_DATE
+
+@Stepwise
+@Ignore
```

Comment:
If we ignore it then maybe let's remove it? :)

Reply:
I don't mind removing it all together. I had just tried to make a parallel for each piece of the Java-based variant. Want me to cut it out?

Reply:
Yeah let's remove both

Reply:
I've removed the `@Ignored` tests.